build/ci/community/prepare_env.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash -xel
+set +o pipefail
+
+csv_version=`yq eval .channels[0].currentCSV deploy/olm-catalog/ibm-block-csi-operator-community/ibm-block-csi-operator.package.yaml`
```

Comment:
`deploy/olm-catalog/ibm-block-csi-operator-community/ibm-block-csi-operator.package.yaml` can be in a var